src/app/host/[roomId]/page.tsx

```diff
@@ -169,21 +173,54 @@ export default function HostRoom({ params }: { params: { roomId: string } }) {
 		})
 	}
 
+	///////////////////////////////////////////////////////////////////////////////////
+	// TODO remove realUsers state, handleAddRandomUsers, and handleRemoveRandomUsers
+	// when done with development
+	const [realUsers, setRealUsers] = useState<ListenerRes[]>([])
+	function handleAddRandomUsers() {
+		const namesStart = Math.floor(Math.random() * 150)
+		const namesEnd = namesStart + 15
+		const sampleNames = namesArray.slice(namesStart, namesEnd)
+
+		setAllUsersPointsData((prevUsersPoints) => {
+			const newUsers = sampleNames.map((name, index) => {
+				const point = Math.floor(Math.random() * 20)
+				const storyPoint = point > 12 ? -99 : point
+				const userName = `${name}*`
+				const imageNum = allUsersPointsData.length + index
+				const timeStamp = Date.now() + Math.floor(Math.random() * 60000) - 60000
+				// NOTE: 60000ms = 1 minute. Each sample user would get
```

Comment:
praise: thanks for this code comments, they are super useful for others and future-you. good code comments explain they why ^.^

Reply:
When I was doing the video, the comments I already had really helped!! But there were other parts I had to really think about how to explain it. So I added more comments!! I thought it would make your job easier, and mine!! Thanks!!